### PR TITLE
fix: fixing over association attribute value

### PR DESCRIPTION
**Pull request by diego-antonelli**

### CODE REVIEW

File: src/BootstrapTooltipContext.js [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
 
 export default declare("BootstrapTooltip.widget.BootstrapTooltipContext", [_bootstrapTooltipWidget], {
     _contextObj: null,
+    _attribute: "",
+
+    postCreate: function () {
+        const attributeSplit = this.tooltipMessageAttribute.split("/");
+        if (attributeSplit.length === 1) {
+            this._attribute = this.tooltipMessageAttribute;
+        } else{
```
Comment (jordanch) [inline src/BootstrapTooltipContext.js]: nit: space

File: src/BootstrapTooltipContext.js [hunk 2]
```diff
@@ -37,23 +49,27 @@ export default declare("BootstrapTooltip.widget.BootstrapTooltipContext", [_boot
         if (this._contextObj) {
             this.subscribe({
                 guid: this._contextObj.getGuid(),
-                callback: () => this.setTooltipTextAttribute()
+                callback: lang.hitch(this, (guid) => {
```
Comment (jordanch) [inline src/BootstrapTooltipContext.js]: What's the purpose here with `hitch` and unused `guid`? Seems like this isn't necessary.

File: src/BootstrapTooltipContext.js [hunk 3]
```diff
@@ -37,23 +49,27 @@ export default declare("BootstrapTooltip.widget.BootstrapTooltipContext", [_boot
...
+                    this.setTooltipTextAttribute()
+                })
             });
 
             this.subscribe({
                 guid: this._contextObj.getGuid(),
-                attr: this.tooltipMessageAttribute,
-                callback: () => this.setTooltipTextAttribute()
+                attr: this._attribute,
+                callback: lang.hitch(this, (guid, attr, attrValue) => {
```
Comment (jordanch) [inline src/BootstrapTooltipContext.js]: What's the purpose here with `hitch` and unused parameters? Seems like this isn't necessary.

**Review by jordanch — CHANGES_REQUESTED** [review]

File: src/BootstrapTooltipContext.js (hunk 2 above)
Comment (diego-antonelli) [reply src/BootstrapTooltipContext.js]: https://dojotoolkit.org/reference-guide/1.10/dojo/_base/lang.html#dojo-base-lang-hitch

File: src/BootstrapTooltipContext.js (hunk 3 above)
Comment (diego-antonelli) [reply src/BootstrapTooltipContext.js]: https://dojotoolkit.org/reference-guide/1.10/dojo/_base/lang.html#dojo-base-lang-hitch

File: src/BootstrapTooltipContext.js (hunk 2 above)
Comment (diego-antonelli) [reply src/BootstrapTooltipContext.js]: Arguments removed

File: src/BootstrapTooltipContext.js (hunk 3 above)
Comment (diego-antonelli) [reply src/BootstrapTooltipContext.js]: Arguments removed

File: src/BootstrapTooltipContext.js (hunk 1 above)
Comment (diego-antonelli) [reply src/BootstrapTooltipContext.js]: done

File: src/BootstrapTooltipContext.js [hunk 4]
```diff
@@ -37,23 +49,27 @@ export default declare("BootstrapTooltip.widget.BootstrapTooltipContext", [_boot
         if (this._contextObj) {
             this.subscribe({
                 guid: this._contextObj.getGuid(),
-                callback: () => this.setTooltipTextAttribute()
+                callback: lang.hitch(this, () => {
```
Comment (jordanch) [inline src/BootstrapTooltipContext.js]: Like below, the semi-colon is missing and this could be an arrow function without body brackets.

**Review by jordanch — CHANGES_REQUESTED** [review]

File: src/BootstrapTooltipContext.js (hunk 4 above)
Comment (diego-antonelli) [reply src/BootstrapTooltipContext.js]: Done

**Review by jordanch — APPROVED** [review]

**Review by leonardomendix — APPROVED** [review]

**Pull request merged**